src/base/bittorrent/addtorrentparams.h

```diff
@@ -35,13 +35,12 @@
 #include <QVector>
 
 #include "base/tagset.h"
+#include "downloadpriority.h"
 #include "torrent.h"
 #include "torrentcontentlayout.h"
 
 namespace BitTorrent
 {
-    enum class DownloadPriority;
```

Comment:
why this change?

Reply:
Otherwise this declaration overrides the one in `downloadpriority.h` and the value `::Normal` is not available

Reply:
I think you should add `#include "downloadpriority.h"` to session.cpp (torrentimpl.cpp already includes it) instead of this change, would that work?

Reply:
Done